src/libraries/System.Private.CoreLib/src/System/Numerics/IExponentialFunctions.cs

```diff
@@ -16,7 +16,7 @@ public interface IExponentialFunctions<TSelf>
         /// <summary>Computes <c>E</c> raised to a given power and subtracts one.</summary>
         /// <param name="x">The power to which <c>E</c> is raised.</param>
         /// <returns><c>E<sup><paramref name="x" /></sup> - 1</c></returns>
-        static abstract TSelf ExpM1(TSelf x);
+        static virtual TSelf ExpM1(TSelf x) => TSelf.Exp(x) - TSelf.One;
```

Comment:
What is this used for?

Reply:
To simplify the amount of work users need to do for their own types. Its still recommended users provide their own implementation for perf or for higher accuracy, but its not required they do so.

Reply:
Oh, sorry, to clarify, I'm wondering what the use case of ExpM1 is.

Reply:
Ah, I misunderstood.

Binary floating-point can only represent multiples of powers of two and so there is inherent error in the represented result. `ExpM1` is a special function that can (but is not required to) provide additional accuracy over the normal `Exp(x) - 1` sequence you'd do and so it can produce a more correct result overall.